cmd/inspect.go

```diff
@@ -76,10 +79,6 @@ var inspectCmd = &cobra.Command{
 			return fmt.Errorf("failed to list containers: %s", err)
 		}
 
-		if len(containers) == 0 {
-			log.Println("no containers found")
-			return nil
-		}
```

Comment:
we need to find a way of not printing an empty table if no containers are found

right now it prints an empty one, and before we used "no containers found"

Reply:
The issue was, that this was printed even in the json case ... which is bad when json output is expected.
What exactly is the issue with the empty table?

Reply:
it just doesn't make a lot of sense to me printing an empty table. I haven't seen any other tool doing that. Gives an impression that something

we need to make sure it is not printed when format is set to json and printed when default output. I can have a look at it if you have more interesting things to have a crack on

Reply:
I'd personally prefere a stable output method. But nevermind.
Check out the new commit. 
https://github.com/srl-labs/containerlab/pull/887/commits/bcc6a2d213394fabb827e14600e593bb199fc4a3

Reply:
why revert the test based on json?
lets keep both?! Just to make sure json output is also fine in the zero container case.

Reply:
valid point. need to get back